src/chocolatey.resources/helpers/functions/Get-WebHeaders.ps1

```diff
@@ -54,8 +57,9 @@ Get-WebFile
 #>
 param(
   [parameter(Mandatory=$false, Position=0)][string] $url = '',
-  [parameter(Mandatory=$false, Position=1)][string] $userAgent = 'chocolatey command line',
-  [parameter(ValueFromRemainingArguments = $true)][Object[]] $ignoredArguments
+  [parameter(Mandatory=$false, Position=1)][string] $userAgent,
+  [parameter(ValueFromRemainingArguments = $true)][Object[]] $ignoredArguments,
+  [parameter(Mandatory=$false)][hashtable] $options = @{Headers=@{}}

```

Comment:
This needs to be before $ignoredArguments. Those are the last items to be passed.